services/frontend/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.7'
+
+x-frontend: &frontend
+  build:
+    context: .
+    dockerfile: services/frontend/Dockerfile
+  image: frontend
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/frontend
+
+services:
+  frontend-lite:
+    <<: *frontend
+    privileged: true
+    volumes:
+      - ..:/govuk:delegated
+      - bundle:/usr/local/bundle
+      - ./frontend/chromedriver-version:/home/build/.chromedriver-helper/.chromedriver-version
```

Comment:
We should be able to remove these volumes

Reply:
Yes please :-). It caused me some pain in another service I was trying to update - couldn't figure out why the volumes weren't doing what I told them too.

Reply:
Done